tex/rules.tex

```diff
@@ -1114,6 +1114,10 @@ \subsection{Preamble}
 
 \item \underline{\href{http://schweikert-shop.he-hosting.de/index.php?cat=2259&lang=ENG&product=93011}{http://schweikert-shop.he-hosting.de/index.php?cat=2259\&lang=ENG\&product=93011}}
 
+    \added[id=TC]{Note that since the e-shop may also send you a semi-glossy ball
```

Comment:
@gngrbrdmn I am not exactly sure about the wording here -- please feel free to change any update that may improve clarity.

Thanks!